packages/suite/src/middlewares/wallet/coinjoinMiddleware.ts

```diff
@@ -75,10 +88,8 @@ export const coinjoinMiddleware =
                     api.dispatch(coinjoinAccountActions.pauseCoinjoinSession(account.key, true));
                 }
             } else if (
-                account &&
-                selectedAccountPrevStatus === 'out-of-sync' &&
                 selectedAccountNextStatus === 'ready' &&
-                isCoinjoinResumeAllowed
+                !checkIsCoinjoinResumeBlocked(state)
```

Comment:
This is run before changeing the status so what happens when account is `out-of-sync` and next status is going to be `ready` but at this point it is sill `out-of-sync`?
I think `checkIsCoinjoinResumeBlocked` will return true and paused accounts are not going to be resumed.

Reply:
You are right, I removed this condition from the function so that it is not checked here. 766664de83cfe6631e86b13b3afddca5dbe30f93